app/controllers/concerns/graphql_helper.rb

```diff
@@ -209,7 +209,7 @@ def my_collection_create_artwork_mutation_params(submission)
       editionSize: submission.edition_size,
       externalImageUrls:
         submission.images.map do |image|
-          image.original_image&.split('?')&.first
+          image.image_urls["large"]
```

Comment:
**Q:** is the `image_urls` available immediately always?

Reply:
This is a very good question. Trying to find it out right now to check if this solution would work.

Reply:
thanks 🙏

Reply:
I've checked and they are immediately available because the images get added earlier in the flow. I'd suggest to merge this change now and check if works on Staging before merging the change.

Reply:
awesome